client/components/jetpack/daily-backup-status/status-card/backup-in-progress.tsx

```diff
@@ -26,6 +28,10 @@ const BackupInProgress: React.FC< Props > = ( { percent, inProgressDate, lastBac
 		? getDisplayDate( lastBackupDate, false )
 		: undefined;
 
+	useEffect( () => {
+		recordLogRocketEvent( 'calypso_jetpack_backup_in_progress' );
+	} );
```

Comment:
Another nitpick:  I know not including the useEffect dependency array (useEffect second argument) defaults to an empty array, still I think it's more explicit, clear, and readable to include it. Just my opinion. Change or not change at your discretion. 🙂

Reply:
You are right! It is better to be explicit. I just added the empty dependency array for both cases. Thanks a lot!